apps/change-log.md

```diff
@@ -14,6 +14,28 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2023-04-13
```

Comment:
Did you intentionally use yesterday's date?

Reply:
@ywolpers Yes, because it is the release date of the public roll-out of the current software version. This is more important than the actual date that the change log entry will be merged.